charts/guac/tests/osv-certifier_deployment_test.yaml

```diff
@@ -35,7 +35,7 @@ tests:
           value: "-c"
       - equal:
           path: spec.template.spec.containers[0].command[2]
-          value: "/opt/guac/guacone certifier osv --poll"
+          value: "/opt/guac/guaccollect osv"
```

Comment:
note: this will only scan the packages once but that is ok for testing. We can change that later.

Reply:
how does the config look like for continuous/periodic scanning?